product_docs/docs/epas/15/installing/linux_x86_64/epas_centos_7.mdx

```diff
@@ -46,39 +46,72 @@ Installing the server package creates an operating system user named enterprised
 
 This section steps you through getting started with your cluster including logging in, ensuring the installation and initial configuration was successful, connecting to your cluster, and creating the user password.
 
+First you need to initialize and start the database cluster.
+
 ```shell
-# Initialize the database cluster
-PGSETUP_INITDB_OPTIONS="-E UTF-8" /usr/edb/as15/bin/edb-as-15-setup initdb
+sudo PGSETUP_INITDB_OPTIONS="-E UTF-8" /usr/edb/as15/bin/edb-as-15-setup initdb
+
+sudo systemctl start edb-as-15
+```
 
-# Start the database cluster
-systemctl start edb-as-15
+To work in your cluster, login as the enterprisedb user. Connect to the database server using the psql command line client.
 
-# To work in your cluster, login as the enterprisedb user
-su - enterprisedb
+```shell
+sudo su - enterprisedb
 
-# Connect to the database server using the psql command line client
 psql edb
+```
+
+The server runs with the `peer` or `ident` permission by default. You can change the authentication method by modifying the `pg_hba.conf` file.
+
+Before changing the authentication method, assign a password to the database superuser, enterprisedb. For more information on changing the authentication, see [modifying the pg_hba.conf file](/epas/latest/epas_guide/03_database_administration/01_configuration_parameters/01_setting_new_parameters/#modifying-the-pg_hbaconf-file).
 
-# Assign a password to the database superuser the enterprisedb
+```sql
 ALTER ROLE enterprisedb IDENTIFIED BY password;
+```
+
+## Experiment
+
+Now you are ready to create and connect to a database, create a table, insert data in a table, and view the data from the table.
+
+This example shows the commands you run. The following example includes the output you’ll see.
 
-# Create a database (named hr)
+```sql
 CREATE DATABASE hr;
 
-# Connect to the new database and create a table (named dept)
 \c hr
+
 CREATE TABLE public.dept (deptno numeric(2) NOT NULL CONSTRAINT dept_pk
 PRIMARY KEY, dname varchar(14) CONSTRAINT dept_dname_uq UNIQUE, loc
 varchar(13));
 
-# Add data to the table
 INSERT INTO dept VALUES (10,'ACCOUNTING','NEW YORK');
 INSERT into dept VALUES (20,'RESEARCH','DALLAS');
 
-# You can use simple SQL commands to query the database and retrieve
-# information about the data you have added to the table
 SELECT * FROM dept;
-__OUTPUT__
+```
+
+This example includes the output of the commands so you can cross-check the output when you run the commands. Don't copy commands from this example as it includes the output and would result in errors.
```

Comment:
@josh-heyer, I added this second example with outputs as I am getting the output at execution of each command and not at last after execution of all commands. Do you agree with this approach? or you have any other recommendation?

Reply:
I don't like it; it's verbose, and it separates the commands that will be executed from their output, which is the whole point. 

I would say, if the output is too extensive to be comfortable... Break it up. This is a demo section - there is no particular need to have it easy to copy-paste en masse if that interferes with readability. Each command or logically-grouped set of commands can then have their own `__OUTPUT__` section without repetition.